elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -88,5 +98,27 @@ $(function() {
             <li class="{{block_formatting}}" style="margin-left: {{indent}}px">{{{inner_html}}}</li>
         {{/question_solution}}
         </ul>
+        {{#has-distractors}}
+        </br>
+        <ul class="list-group">
+        <li class="list-group-item list-group-item-danger"><strong>
+        Incorrect Blocks (these blocks should not be included in your solution):</strong></li>
+        </ul>
+        <ul class="list-group answer_table" style="list-style-type:none;">
+        {{#distractor-feedback}}
+            <li> 
+                <div class="{{block_formatting}}">{{{inner_html}}}</div> 
+                {{#has-distractor-feedback}}
+                <div style="margin: 10px" > 
+                    <div style="display: flex; align-items: center;">
```

Comment:
get rid of the style here to fix the issue noted in my other comment (https://github.com/PrairieLearn/PrairieLearn/pull/5667#issuecomment-1239756742)